package.json

```diff
@@ -71,7 +71,7 @@
     "devtools-launchpad": "^0.0.119",
     "devtools-linters": "^0.0.4",
     "devtools-reps": "^0.22.0",
-    "devtools-source-map": "^0.16.0",
+    "devtools-source-map": "0.16.0",
```

Comment:
should we add a comment saying the version number will be _ignored_ because we use workspaces ?

Reply:
right 🤦‍♂️